Library/Homebrew/cmd/style.rb

```diff
@@ -1,3 +1,18 @@
+#:  * `style` [`--fix`] <targets>
+#:    Check formulae or files
```

Comment:
What are we checking for? The current wording is a bit too vague. Also misses a trailing full stop.